components/infobox/wikis/apexlegends/infobox_map_custom.lua

```diff
@@ -0,0 +1,115 @@
+---
+-- @Liquipedia
+-- wiki=apexlegends
+-- page=Module:Infobox/Map/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Map = require('Module:Infobox/Map')
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local String = require('Module:StringUtils')
+local Title = require('Module:Infobox/Widget/Title')
+local Center = require('Module:Infobox/Widget/Center')
+
+local CustomMap = Class.new()
+local CustomInjector = Class.new(Injector)
+
+local _args
+local _map
+
+function CustomMap.run(frame)
+	local map = Map(frame)
+	_map = map
+	_args = map.args
+	map.addToLpdb = CustomMap.addToLpdb
+	map.createWidgetInjector = CustomMap.createWidgetInjector
+	return map:createInfobox(frame)
+end
+
+function CustomMap:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:addCustomCells(widgets)
+	table.insert(widgets, Cell{
+		name = 'Game Mode(s)',
+		content = {_args.gamemode}
+	})
+	local spantext = ''
+	if String.isNotEmpty(_args.spanstart) then
+		spantext = spantext .. ' - '
+	end
+	table.insert(widgets, Cell{
+		name = 'Played in ALGS',
+		content = {(_args.spanstart or '<i><b>Not </b></i>') .. spantext .. (_args.spanend or '<i><b>Currently</b></i>')}
+	})
+	if String.isNotEmpty(_args.ring) then
+		local ringTable = mw.html.create('table')
+			:addClass('wikitable wikitable-striped wikitable-bordered')
+			:css('width', '325px')
+		CustomMap:_createRingTable(ringTable)
+		for _, rings in ipairs(_map:getAllArgsForBase(_args, 'ring')) do
+			CustomMap:_createRingTable(ringTable, rings)
+		end
+		table.insert(widgets, Title{name = 'Ring Information'})
+		table.insert(widgets, Center{content = {tostring(ringTable)}})
+	end
+	return widgets
+end
+
+function CustomMap:_createRingTable(ringTable, content)
+	local row = ringTable:tag('tr')
+
+	if not content then
+		row:tag('th')
```

Comment:
```suggestion
		row
 			:tag('th'):wikitext('Ring'):done()
 			:tag('th'):wikitext('Wait(s)'):done()
 			:tag('th'):wikitext('Close<br>Time(s)'):done()
 			:tag('th'):wikitext('Damage<br>per tick'):done()
 			:tag('th'):wikitext('End Diameter (m)'):done()
```
on mobile, so can not span the comment, is intended to go until row 81 (right before the else)